torch/_inductor/fx_passes/post_grad.py

```diff
@@ -81,8 +82,27 @@ def visit(other_node):
             node.prepend(other_node)
 
     seen_nodes = set()
+
+    # only reorder nodes before the first copy_ in the graph.
```

Comment:
Since we iterate in reverse order, wouldn't this do after the first copy?  Which might include other copies...

Reply:
We are find the first copy not in reversed iteration, maybe last_copy would be clearer ?